### DISCUSSION TITLE
Copilot for team

### DISCUSSION THREAD

**Discussion by Energytechglobal** [community/community#32775] [Copilot Conversations] [answered]
Hi,

I am Prasanthi from EnergyTech Global, Hyderabad.
We are looking for co-pilot for team. Please let me know how to procure the license for a team.

Thanks,
Prasanthi

**Comment by iamcjwilliams** [comment, marked as answer]
@Energytechglobal Currently, Copilot is only available for individual user accounts and organizations aren't able to purchase/manage Copilot for their members just yet. However, you can sign up on the waitlist here: https://github.com/features/copilot/org_signup

**Comment by AdriaRios** [comment]
Hello!

I am trying to fill this form. I got several errors telling that I have to fill all the fields (were already filled). Now I am getting a message saying that I already sent the form.

I didn't received a confirmation email of something, can someone help me to contact support and check if the request were successfully made?

Thank you!

**Reply by iamcjwilliams** [reply]
@AdriaRios The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted
